JoysOfEfficiency/Huds/FishingProbabilitiesBox.cs

```diff
@@ -106,12 +106,12 @@ public static void PrintFishingInfo()
 
             LocationData thisLocData = Game1.currentLocation.GetData();
             String jThisLocData = JsonSerializer.Serialize(thisLocData, new JsonSerializerOptions { WriteIndented = true });
-            Logger.Info($"This LocationData: {jThisLocData}");
+            //Logger.Info($"This LocationData: {jThisLocData}");
```

Comment:
Leaving commented-out code around is not generally advisable.

Perhaps instead consider reducing them from `Info` to `Trace` so they are not printed to console unless you're using SMAPI for Developers, or delete them / tie them to a compile-time switch as they will still explode the log file even under `Trace` (see [log level documentation](https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Logging#Log_levels)).

Considering the massive quantity of information in these, I'd probably opt for enabling them only in a debug build. It is unkind to performance and the end user's disk to log all that during gameplay.

https://github.com/Hackswell/JoysOfEfficiency/blob/1861eeb9f0520abf3859ddde18789b6b6bbfa3dc/JoysOfEfficiency/Utils/Logger.cs#L23-L31